README.md

```diff
@@ -10,3 +10,40 @@ To run the scraper, you can use the run script by typing `sh run_scraper.sh` int
 
 ## Running the API
 The best way to run the API right now is to run the command `FLASK_APP="app.py" FLASK_ENV=development flask run;`. Note that this is not the best way to run the scraper at this time.
+
+## Development
+
+We use CircleCI to lint the code and run tests in this repository, but you can (and should!) also run tests locally.
+
+The commands described below should all be run from within the virtual environment you’ve created for this project. If you used `install.sh` to get set up, you’ll need to activate your virtual environment before running them with the command:
+
+```sh
+$ source env/bin/activate
+```
+
+If you manage your environments differently (e.g. with Conda or Pyenv-Virtualenv), use whatever method you normally do to set up your environment.
+
+### Tests
+
+You can run tests using `pytest` like so:
+
+```sh
+# In the root directory of the project:
+$ python -m pytest -v .
```

Comment:
I got a couple warnings from running pytest on root. Are these ok?
```
env/lib/python3.7/site-packages/html5lib/_trie/_base.py:3
  /Users/elainelaguerta/OpenSourceProjects/data-covid19-sfbayarea/env/lib/python3.7/site-packages/html5lib/_trie/_base.py:3: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import Mapping

covid19_sfbayarea/news/utils.py:42
  /Users/elainelaguerta/OpenSourceProjects/data-covid19-sfbayarea/covid19_sfbayarea/news/utils.py:42: DeprecationWarning: invalid escape sequence \?
    b'<?xml\\s[^>]*encoding=[\'"]([^\'"]+)[\'"].*\?>',
```

Reply:
Well, they’re both warnings, so technically fine. 😉 

But anyway, the first one is coming from `env/lib/python3.7/site-packages/html5lib/_trie/_base.py`, i.e. it’s coming from a library we depend on, not from us. (That said, [it looks like it’s been fixed in v1.1](https://github.com/html5lib/html5lib-python/blob/master/CHANGES.rst#11), which just came out last week, so we should consider upgrading. I think that’s out of scope for this PR, though.)

The second one is new in Python 3.8, but is a good warning I should probably fix.

Reply:
Actually, that’s not new in 3.8, but I’m not getting it locally for some reason. Odddddddd.

Reply:
Ah, Pytest was being helpful by not repeating warnings it had already shown me on files I hadn’t edited. Ha. Anyway, fixed.